src/sentry/static/sentry/app/api.jsx

```diff
@@ -191,13 +191,12 @@ export class Client {
     return this.activeRequests[id];
   }
 
-  requestPromise(path, options = {}) {
+  requestPromise(path, {includeAllArgs, ...options} = {}) {
     return new Promise((resolve, reject) => {
       this.request(path, {
         ...options,
-        success: (data, ...args) => {
-          // This fails if we need jqXhr :(
-          resolve(data);
+        success: (data, successString, allArgs) => {
```

Comment:
```suggestion
        success: (data, ...args) => {
```

And then you can `resolve([data, ...args])` - this way you don't need to worry about naming and order

Reply:
do we really need successString? it's pretty useless, it's a string that says "success", idk if we'd run into a problem like this again for that. 
is it just better practice to do it this way? it seems like it's unnecessary info we can just drop.